From 9894ef8e8ff897ab11b5b8e66a16f1702c8fc5ce Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Thu, 30 Jun 2022 15:15:30 +0200
Subject: [PATCH 2/6] make test: ignore the existing `test/` and run the tests
 already

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 Makefile.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Makefile.in b/Makefile.in
index 4e25ef9..1caeeb1 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -179,6 +179,8 @@ $(ARCHDIR)/%.lo: $(SRCDIR)/$(ARCHDIR)/%.c
 test: all
 	$(MAKE) -C test
 
+.PHONY: test
+
 infcover.o: $(SRCDIR)/test/infcover.c zlib$(SUFFIX).h zconf$(SUFFIX).h zlib_name_mangling$(SUFFIX).h
 	$(CC) $(CFLAGS) $(INCLUDES) -c -o $@ $(SRCDIR)/test/infcover.c
 
-- 
2.37.0.rc2.windows.1

