From 82e62b26d128526c9c0cc7e9a21a5d8b1de916b6 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Thu, 30 Jun 2022 15:26:07 +0200
Subject: [PATCH 3/6] Fix test binaries' dependency on libz

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 test/Makefile.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/Makefile.in b/test/Makefile.in
index 4cd1399..a71a3c9 100644
--- a/test/Makefile.in
+++ b/test/Makefile.in
@@ -56,7 +56,7 @@ ghtests: testGH-361 testGH-364 testGH-751 testGH-1235
 testGH-361:
 	$(QEMU_RUN) ../minigzip$(EXE) -4 <$(SRCDIR)/GH-361/test.txt >/dev/null
 
-switchlevels$(EXE): $(SRCDIR)/switchlevels.c
+switchlevels$(EXE): $(SRCDIR)/switchlevels.c ../$(LIBNAME).a
 	$(CC) $(CFLAGS) -I.. -I$(SRCTOP) -o $@ $< $(TEST_LDFLAGS)
 
 .PHONY: testGH-364
@@ -67,7 +67,7 @@ testGH-364: switchlevels$(EXE)
 testGH-751:
 	$(QEMU_RUN) ../minigzip$(EXE) <$(SRCDIR)/GH-751/test.txt | $(QEMU_RUN) ../minigzip$(EXE) -d >/dev/null
 
-gh1235$(EXE): $(SRCDIR)/gh1235.c
+gh1235$(EXE): $(SRCDIR)/gh1235.c ../$(LIBNAME).a
 	$(CC) $(CFLAGS) -I.. -I$(SRCTOP) -o $@ $< $(TEST_LDFLAGS)
 
 .PHONY: testGH-1235
-- 
2.37.0.rc2.windows.1

